src/components/SearchResults.js

```diff
@@ -88,8 +99,8 @@ export default class SearchResults extends React.Component<SearchResultsDefaultP
 
   renderResults() {
     const searcher = this.context.searcher;
-    const response = searcher.state.response;
-    const offset = searcher.state.resultsOffset;
+    const response = this.props.response !== null ? this.props.response : searcher.state.response;
+    const offset = this.props.response ? this.props.offset : searcher.state.resultsOffset;
```

Comment:
Make `offset` optional for backwards compatibility.

Reply:
Changed to:
`const offset = this.props.response && this.props.offset ? this.props.offset : searcher.state.resultsOffset;`

Is this what you meant?